docs/FAQ.md

```diff
@@ -0,0 +1,32 @@
+This page is for frequent asked questions and answers.
+
+
+### tmp folder fulled
+NNICTL will use tmp folder as a temporary folder to copy files under codeDir when executing experimentation creation. 
+When met errors like below, try clean up **tmp** folder first.
+> OSError: [Errno 28] No space left on device
+
+### Cannot get trials' metrics in OpenPAI mode
+In OpenPAI training mode, we start a rest server which listens on 51189 port in NNIManager to receive metrcis reported from trials running in OpenPAI cluster. If you didn't see any metrics from WebUI in OpenPAI mode, check your machine where NNIManager runs on to make sure 51189 port is turned on in the firewall rule. 
```

Comment:
"nni_manager" and "NNIManager" are both in this file. I think you could choose one.

Reply:
ok, will fix